src/test/kotlin/apply/domain/evaluationtarget/EvaluationTargetRepositoryTest.kt

```diff
@@ -66,4 +66,22 @@ class EvaluationTargetRepositoryTest(
 
         assertThat(evaluationTargetRepository.count()).isEqualTo(1)
     }
+
+    @ParameterizedTest
+    @CsvSource(value = ["PASS,2", "FAIL,1"])
```

Comment:
```suggestion
    @CsvSource(value = ["PASS,2", "FAIL,1"])
    @ParameterizedTest
```
변경 가능성이 높은 어노테이션을 맨 위로 위치시키는게 좋을 것 같다고 하셨던 것 같아요!